.prettierignore

```diff
@@ -0,0 +1 @@
+extension-manifest-v2/**/*
```

Comment:
Prettier in v2 is in conflict with eslint rules. I don't know a better option for now than disabling it there.